mono/metadata/threads.h

```diff
@@ -54,7 +54,7 @@ MONO_API void mono_threads_request_thread_dump (void);
 
 MONO_API mono_bool mono_thread_is_foreign (MonoThread *thread);
 
-extern MONO_API void mono_thread_detach_if_exiting (void);
+extern MONO_API int32_t mono_thread_detach_if_exiting (void);
```

Comment:
Because we don't include `glib.h`, and the equivalent to `gboolean` is `int32_t` (https://github.com/mono/mono/blob/master/eglib/src/glib.h#L73)